conf/messages

```diff
@@ -1,2 +1,5 @@
 service.name = customs-update-eori-admin-frontend
 service.text = This is your new service
+
+Do_you_want_to_replace_an_existing_EORI_number_or_cancel_subscriptions_to_HMRC_services=Do you want to replace an existing EORI number or cancel subscriptions \n to HMRC services?
```

Comment:
Also can we give better name to that like @Messages("eoriAction.questionForReplaceOrCancel") or better one if you have on your mind :)